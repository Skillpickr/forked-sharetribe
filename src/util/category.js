const Categories = {
  creative: 'creative',
  performance: 'performance',
  knowledge: 'knowledge',
  audioProd: 'audio-prod',
  mediaProd: 'media-prod'
}

const Skills = {
  photographer: 'photographer',
  videographer: 'videographer',
  dj: 'dj',
  musician: 'musician',
  band: 'band',
  makeupArtist: 'makeupArtist',
  dancer: 'dancer',
  miscellaneous: 'miscellaneous'
}
export { Categories, Skills }
