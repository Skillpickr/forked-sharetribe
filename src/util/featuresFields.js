const InputFieldsType = {}

const CheckboxFieldsType = {
  photographerTypeKey: 'photographerType',
  djTypeKey: 'djType',
  musicianTypeKey: 'musicianType',
  musicianGenreKey: 'musicianGenre',
  bandGenreKey: 'bandGenre',
  bandTypeKey: 'bandType'
}

const DropdownFieldsType = {
  musicianSoloKey: 'musicianSoloType',
  ownStudioKey: 'ownStudio',
  soundLightExpKey: 'soundLightExp',
  djGearForPlayingKey: 'djGearForPlaying',
  songRequestKey: 'songRequest'
}

export { InputFieldsType, CheckboxFieldsType, DropdownFieldsType }
