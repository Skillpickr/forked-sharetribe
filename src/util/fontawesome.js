// import the library
import { library, config } from '@fortawesome/fontawesome-svg-core'

// import your icons
import { fas, faCircleQuestion, faMagnifyingGlass } from '@fortawesome/free-solid-svg-icons'

import { fab } from '@fortawesome/free-brands-svg-icons'

config.autoAddCss = false
library.add(
  fas,
  fab,
  faCircleQuestion,
  faMagnifyingGlass
  // more icons go here
)
