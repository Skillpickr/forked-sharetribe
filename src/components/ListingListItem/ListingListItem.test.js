import React from 'react'
import { renderShallow } from '../../util/test-helpers'
import { createUser, createListing, fakeIntl } from '../../util/test-data'
import { shallow } from 'enzyme'
import { ListingListItem } from '../index'

const intl = {
  formatMessage: jest.fn()
}

describe('ListingListItem', () => {
  it('matches snapshot', () => {
    const listing = createListing('listing1', {}, { author: createUser('user1') })
    const tree = renderShallow(<ListingListItem listing={listing} intl={fakeIntl} />)
    expect(tree).toMatchSnapshot()
  })
  it('should render the component', () => {
    const wrapper = shallow(
      <ListingListItem
        className="test-class"
        rootClassName="test-root-class"
        intl={intl}
        listing={{}}
        renderSizes=""
        filtersConfig={{}}
        setActiveListing={jest.fn()}
      />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
