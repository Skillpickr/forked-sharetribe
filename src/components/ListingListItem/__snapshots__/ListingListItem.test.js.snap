// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListingListItem matches snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`ListingListItem should render the component 1`] = `ShallowWrapper {}`;
